src/fluree/server/consensus.clj

```diff
@@ -22,3 +26,20 @@
   (log/trace "queue-new-transaction:" txn)
   (let [event-params (events/commit-transaction ledger-id tx-id txn opts)]
     (-queue-new-transaction transactor event-params)))
+
+(defn queue-drop-ledger
+  [transactor ledger-id]
+  (let [event-params (events/drop-ledger ledger-id)]
+    (-queue-drop-ledger transactor event-params)))
+
+(defn monitor-consensus
```

Comment:
I don't think this function should go here. The consensus component should not be concerned with putting messages on the watcher. That's why this function was within the handler namespace hierarchy to begin with because that's the integration point that ties the independent components together. If you're worried about requiring one handler in another, then add it to the shared handler namespace.